src/FileService.Broker/Consumers/AddImageConsumer.cs

```diff
@@ -1,15 +1,44 @@
 ﻿using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.FileService.Business.Interfaces;
+using LT.DigitalOffice.FileService.Mappers.RequestMappers.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
 using MassTransit;
-using System;
+using Microsoft.AspNetCore.Http;
 using System.Threading.Tasks;
 
 namespace LT.DigitalOffice.FileService.Broker.Consumers
 {
     public class AddImageConsumer : IConsumer<IAddImageRequest>
     {
-        public Task Consume(ConsumeContext<IAddImageRequest> context)
+        private readonly IAddNewImageCommand _command;
+        private readonly IImageRequestMapper _mapper;
+
+        private object GetImage(IAddImageRequest request)
+        {
+            var imageRequest = _mapper.Map(request);
+
+            var imageId = _command.Execute(imageRequest, request.UserId);
```

Comment:
but the logic of creating an image should be the same, so I now use this command, it allows you not to create copies of existing classes, and also confirms that we will not have any differences in the logic of creating images through the broker and through the request. I asked Pasha about this and we came to the conclusion that this is a normal practice here.

Reply:
This is not normal praсtice, use Mapper in consumer for mapp IAddImageRequest => ImageRequest. ImageRequest used for outer requests, for fronted. Either they dto map from model to database model or vice versa. Use the practice of isolating different levels of code from each other. And if we suddenly have to change the logic of processing the received message, you will change the command?

Reply:
This is the command for adding, I guess if format adding change we will change mappers and command

Reply:
look on the other side. The first solution seems that it is necessary to describe the entire solution either in the consumer or in the new command used in it, write a mapper for the request. And what we will have, there will be two commands with the same code, exactly the same, and two identical mappers. And if something changes in one, I think it will force you to change the other, since the logic of adding should be identical.